nixos/modules/services/security/vaultwarden/backup.sh

```diff
@@ -12,6 +12,10 @@ if [[ ! -f "$DATA_FOLDER"/db.sqlite3 ]]; then
 fi
 
 sqlite3 "$DATA_FOLDER"/db.sqlite3 ".backup '$BACKUP_FOLDER/db.sqlite3'"
-cp "$DATA_FOLDER"/rsa_key.{der,pem,pub.der} "$BACKUP_FOLDER"
+cp "$DATA_FOLDER"/rsa_key.* "$BACKUP_FOLDER"
 cp -r "$DATA_FOLDER"/attachments "$BACKUP_FOLDER"
+if [[ -e "$DATA_FOLDER"/config.json ]]; then
```

Comment:
@SuperSandro2000 What are your thoughts on doing the same logic before doing the sqlite dump? Technically this backup script could be used with another backing DB (although a lot less useful). I know the script still continues executing, but it does produce error messages in the logs.

Reply:
sure, why not